test/extended/image_ecosystem/s2i_php.go

```diff
@@ -58,25 +59,33 @@ var _ = g.Describe("[image_ecosystem][php][Slow] hot deploy for openshift php im
 				err = e2e.WaitForEndpoint(oc.KubeFramework().ClientSet, oc.Namespace(), "cakephp-mysql-example")
 				o.Expect(err).NotTo(o.HaveOccurred())
 
-				assertPageCountIs := func(i int) {
+				assertPageRegexpCountIs := func() {
 					_, err := exutil.WaitForPods(oc.KubeClient().Core().Pods(oc.Namespace()), dcLabel, exutil.CheckPodIsRunning, 1, 4*time.Minute)
 					o.ExpectWithOffset(1, err).NotTo(o.HaveOccurred())
 
-					result, err := CheckPageContains(oc, "cakephp-mysql-example", "", pageCountFn(i))
+					result, err := CheckPageRegexp(oc, "cakephp-mysql-example", "", pageRegexpCountFn())
 					o.ExpectWithOffset(1, err).NotTo(o.HaveOccurred())
 					o.ExpectWithOffset(1, result).To(o.BeTrue())
 				}
 
 				g.By("checking page count")
-
-				assertPageCountIs(1)
-				assertPageCountIs(2)
+				assertPageRegexpCountIs()
```

Comment:
doesn't make sense to end this in "Is" if it's not taking an arg.

Also this change seems to lose the logic that confirmed that hitting the page increments the counter.  It would be preferable to retain that checking.

Reply:
Yep made a conscious decision there, as the string parsing logic to capture the incrementing seemed unpleasant enough that it outweighed verifying the incrementing in my mind.

I'll give it a go and push it here, and then we can decided if we want to keep it.